packages/remark-deflist/src/index.spec.js

```diff
@@ -1,85 +1,106 @@
 /* eslint-env jest */
-import deflist from './index'
-import html from 'rehype-stringify'
-import markdown from 'remark-parse'
-import remark2rehype from 'remark-rehype'
-import unified from 'unified'
-
-const strip = ([str]) => str.replace(/\n {6}/g, '\n').replace(/\n {4}$/, '')
-
-const parse = str =>
+import deflist from "./index";
+import html from "rehype-stringify";
+import markdown from "remark-parse";
+import remark2rehype from "remark-rehype";
+import unified from "unified";
+import mdx from "@mdx-js/mdx";
+import mdxUtil from "mdast-util-mdx";
+import syntax from "micromark-extension-mdxjs";
+import remarkSuperSub from "../../remark-supersub/src";
+
+const strip = ([str]) => str.replace(/\n {6}/g, "\n").replace(/\n {4}$/, "");
+
+const parse = (str) =>
   unified()
     .use(markdown)
     .use(deflist)
     .use(remark2rehype)
     .use(html)
     .process(str)
-    .then(data => data.toString())
+    .then(data => data.toString());
+
+const mdxParse = (str) =>
+  mdx(str, {
```

Comment:
I was debating between using a unified pipeline or mdx itself and decided on mdx itself because the output ended up being more of what I expected (like jsx and correct html nodes as opposed to literal braces and unknown nodes becoming <div>s).